crates/motsu/src/lib.rs

```diff
@@ -30,6 +30,28 @@
 //! }
 //! ```
 //!
+//! ### Global Variables
+//!
+//! Motsu allows you to manipulate certain global variables that affect the execution environment:
+//!
+//! #### Chain ID
+//!
+//! You can get and set the Chain ID in tests using the `VMContext` API:
+//!
+//! ```rust,ignore
+//! use motsu::prelude::*;
+//!
+//! #[motsu::test]
+//! fn test_with_custom_chain_id(contract: Contract<MyContract>, alice: Account) {
+//!     // Default chain ID is 42161 (Arbitrum One)
+//!     
+//!     // Set chain ID to 11155111 (Sepolia testnet)
+//!     VMContext::current().set_chain_id(11155111);
+//!     
+//!     // Now any contract code that depends on the chain ID will use this value
+//! }
+//! ```
```

Comment:
```suggestion
//! ### Global Variables
//!
//! Motsu allows you to manipulate certain global variables that affect the
//! execution environment:
//!
//! #### Chain ID
//!
//! You can get and set the Chain ID in tests using the `VMContext` API:
//!
//! ```rust,ignore
//! use motsu::prelude::*;
//!
//! #[motsu::test]
//! fn test_with_custom_chain_id(
//!     contract: Contract<MyContract>,
//!     alice: Account,
//! ) {
//!     // Default chain ID is 42161 (Arbitrum One)
//!
//!     // Set chain ID to 11155111 (Sepolia testnet)
//!     VMContext::current().set_chain_id(11155111);
//!
//!     // Now any contract code that depends on the chain ID will use this
//!     // value
//! }
//! ```
```